test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -1898,6 +2042,10 @@ odp_testinfo_t ipsec_out_suite[] = {
 				  ipsec_check_esp_null_aes_xcbc),
 	ODP_TEST_INFO_CONDITIONAL(test_sa_info,
 				  ipsec_check_esp_aes_cbc_128_sha1),
+	ODP_TEST_INFO_CONDITIONAL(test_out_ipv4_esp_sa_pkt_expiry,
+				  ipsec_check_esp_aes_cbc_128_sha1),
+	ODP_TEST_INFO_CONDITIONAL(test_out_ipv4_esp_sa_byte_expiry,
```

Comment:
It would be good to have SA expiry tests for inbound SAs too, but of course it does not have to be part of this PR. Any plans for it?

Reply:
Will take inbound test as separate PR.